templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_statement_segments_list.html.twig

```diff
@@ -25,7 +25,7 @@
         :recommendation-procedure-ids="JSON.parse('{{ recommendationProcedureIds|json_encode|e('js', 'utf-8') }}')"
         statement-id="{{ statementId }}"
         statement-extern-id="{{ statementExternId }}"
-        statement-form-definitions="{}"
+        :statement-form-definitions="JSON.parse('{{ templateVars.statementFormDefinition|json_encode|e('js', 'utf-8') }}')"
```

Comment:
Thanks for fixing that :) I think we can also remove the TO DO `{#    TP DO: Fill statementFormDefinitions #}` above now.

Reply:
thanks! done ff4bc0d331b55c0bf4bbc03fcb17f895fa6087eb